libcloud/httplib_ssl.py

```diff
@@ -165,7 +179,8 @@ def request(self, method, url, body=None, headers=None, raw=False):
             data=body,
             headers=headers,
             allow_redirects=1,
-            stream=raw
+            stream=raw,
+            verify=self.ca_cert if self.ca_cert is not None else self.verify
```

Comment:
As with other comment, this is actually correct if it is a string. http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification
I see your comment about a list now I didn't try that